### PR TITLE
Store settings for multiple shifting. functionality seems fine. desig…

### PR DESCRIPTION
**Pull request by zamandevs**
…n break when add more shift

### CODE REVIEW

File: includes/Dashboard/Templates/Settings.php [hunk 1]
```diff
@@ -17,9 +17,9 @@ class Settings {
     /**
      * Loading autometically when class initiate
      *
+     * @return void
```
Comment (ediamin) [inline includes/Dashboard/Templates/Settings.php:20]: @zamandevs why did you move this line for every method?

Comment (zamandevs) [reply includes/Dashboard/Templates/Settings.php:20]: didn't notice that. I maintain phpstorm  plugin for Wordpress. when reformatted whole code it occurred.
